src/jmcomic/jm_downloader.py

```
@@ -1,12 +1,81 @@
+        def get_entity(args, kwargs):
+            # 常规位置参数走快路径；只有关键字调用才按参数名取值。
+            if len(args) > 1:
+                return args[1]
+            return kwargs[entity_param]
+
+        if inspect.iscoroutinefunction(func):
+            @wraps(func)
+            async def async_wrapper(*args, **kwargs):
+                entity = get_entity(args, kwargs)
+                detail_call = isinstance(entity, Downloadable)
+                # 顶层 ID 下载负责完整耗时，内部 detail 调用复用同一个计时上下文。
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)